cypress/integration/queryBuilder.spec.js

```diff
@@ -236,13 +249,15 @@ describe('test query builder', () => {
       // 'next' button should bring user to share panel
       cy.get('[data-cy=next]').click();
       cy.contains('Share Snapshot').should('be.visible');
+      const regexString = `/FROM (${cloudPlatform.datasetName}).variant /`;
       cy.window()
         .its('store')
         .invoke('getState')
         .its('snapshots')
         .its('snapshotRequest')
         .its('joinStatement')
-        .should('match', /FROM (V2F_GWAS_Summary_Stats|V2F_GWAS_Summary_Statistics).variant /);
+        .should('match', regexString);
```

Comment:
This tests is now failing - I need to figure out how to have a variable in the regex.

Reply:
Fixed